src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/Issues-and-Incident-management-and-response.mdx

```diff
@@ -28,9 +28,11 @@ import accountsIssueSummary from 'images/accounts_screenshot-full_issue-summary.
 import accountsIssueFeedFilterIcon from 'images/accounts_screenshot-crop_issue-feed-filter-icon.webp'
 
 
-## Issues feed [#issues-feed]
+Use New Relic to track and respond to your issues using issues feeds, postmortem reports, and more. 
```

Comment:
I think it would be helpful to flesh this out a bit, even if it's just basic or maybe obvious stuff. Just a sentence or two that gives value prop or goes over more specifics of the document just provides a smoother transition to the meat of the doc I think.